mona_sdk/client.py

```diff
@@ -763,13 +785,21 @@ def validate_config(
         A wrapper function for "Validate Config" REST endpoint. view full documentation
         here: https://docs.monalabs.io/docs/validate-config-via-rest-api
         """
-        return self._app_server_request(
+        app_server_response = self._app_server_request(
             "validate_config",
             data={
                 "config": config,
                 "list_of_context_ids": list_of_context_ids,
                 "latest_amount": latest_amount,
             },
+        )["response_data"]
+
+        return (
+            self._handle_service_error(
+                (list(app_server_response["issues"].values())[0]).__str__()
+            )
+            if "issues" in app_server_response
+            else get_dict_result(True, "config is valid", None)
```

Comment:
nit - the data in this case - where its not an error - should probably be True rather than a string - given its an api used by machines... I expect the returned data here to be true instead of string...